src/network/tcp.py

```diff
@@ -124,6 +127,23 @@ def antiIntersectionDelay(self, initial=False):
                     ' for %.2fs', delay)
                 self.skipUntil = time.time() + delay
 
+    def checkTimeOffsetNotification(self):
+        """
+        Check if we have connected to too many nodes which have too high
+        time offset from us
+        """
+        if BMProto.timeOffsetWrongCount > \
+                maximumTimeOffsetWrongCount and \
+                not self.fullyEstablished:
+            UISignalQueue.put((
+                'updateStatusBar',
+                # pylint: disable=protected-access
+                tr._translate(
+                    "MainWindow",
+                    "The time on your computer, %1, may be wrong. "
+                    "Please verify your settings."
+                ).arg(datetime.datetime.now().strftime("%H:%M:%S"))))
```

Comment:
Maybe use `l10n.formatTimestamp()` here, like in [class_singleWorker](../../blob/v0.6/src/class_singleWorker.py#L677)?

Reply:
ok